app/src/main/java/br/com/catbag/gifreduxsample/ui/GifListActivity.java

```diff
@@ -74,6 +65,8 @@ private void bindingViews() {
             withId(R.id.loading, () -> {
                 visibility(mGifProgressVisibility);
             });
+
+            if (mAnvilRenderListener != null) mAnvilRenderListener.onAnvilRendered();
```

Comment:
Now that we have a custom Dispatcher that calls `Anvil#render()`, looks easy to move anvilRenderedListener to there. What do do you think? But I don't know if Anvil.render() is a sync or async method.